src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_live_event_scenarios.py

```diff
@@ -63,6 +61,10 @@ def test_live_event_create(self, storage_account_for_create):
         self.assertNotEquals('Stopping', live_event['resourceState'])
         self.assertNotEquals('Stopped', live_event['resourceState'])
 
+        self.cmd('az ams live event stop -a {amsname} -n {liveEventName} -g {rg}')
+
+        self.cmd('az ams live event delete -a {amsname} -n {liveEventName} -g {rg}')
```

Comment:
Make sure you don't get to this point with the live event 'Stopping' instead of 'Stopped', otherwise it will throw an exception and the test will fail.